source/includes/dev/_deletion.md

```diff
@@ -0,0 +1,353 @@
+|-----------------|----------------------------------------------------------|--------|---------------------|
+| userId          | Filter by the id of the user account to be deleted.      | String | any valid user id   |
+| requestorUserId | Filter by the id of the user who requested the deletion. | String | any valid user id   |
+| status          | Filter by the status of the deletion process.            | String | `done` or `pending` |
+
+### Errors for getting all deletions
+
+| Error code | Error message      | Description                                               |
+|------------|--------------------|-----------------------------------------------------------|
+| 401        | Unauthorized       | You need to be logged in to be able to get all deletions. |
+| 403        | Not authorized     | You need to have the `ADMIN` role                         |
+
+## Getting a deletion by id
+
+> Request a deletion by id:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/deletion/<deletion-id> \
+-H "Authorization: Bearer <your-token>"
+```
+
+> Example response:
+
+```json
+{
+  "data": {
+    "type": "deletions",
+    "id": "62bd8d49cae089972ce81039",
+    "attributes": {
+      "userId": "62bd8d49cae089972ce81038",
+      "requestorUserId": "62bd8d49cae089972ce81038",
+      "status": "pending",
+      "datasetsDeleted": false,
+      "layersDeleted": false,
+      "widgetsDeleted": false,
+      "userAccountDeleted": false,
+      "userDataDeleted": false,
+      "graphDataDeleted": false,
+      "collectionsDeleted": false,
+      "favouritesDeleted": false,
+      "vocabulariesDeleted": false,
+      "areasDeleted": false,
+      "storiesDeleted": false,
+      "subscriptionsDeleted": false,
+      "dashboardsDeleted": false,
+      "profilesDeleted": false,
+      "topicsDeleted": false,
+      "createdAt": "2022-06-30T11:47:21.305Z",
+      "updatedAt": "2022-06-30T11:47:21.305Z"
+    }
+  }
+}
+```
+
+This endpoints allows you to retrieve the details of a single deletion record from its id.
+
+### Errors for getting a deletion by id
+
+| Error code | Error message      | Description                                            |
+|------------|--------------------|--------------------------------------------------------|
+| 401        | Unauthorized       | You need to be logged in to be able to get a deletion. |
+| 403        | Not authorized     | You need to have the `ADMIN` role                      |
+| 404        | Deletion not found | There is no deletion with the provided id              |
+
+## Creating a deletion
+
+> Create a deletion for the current user
+
+```shell
+curl -X POST "https://api.resourcewatch.org/v1/deletion" \
+-H "Authorization: Bearer <your-token>"
+```
+
+> Response:
+
+```json
+{
+  "data": {
+    "type": "deletions",
+    "id": "62bd8d49cae089972ce81039",
+    "attributes": {
+      "userId": "62bd8d49cae089972ce81038",
+      "requestorUserId": "62bd8d49cae089972ce81038",
+      "status": "pending",
```

Comment:
`status` seems to be ignored in the route handler and is not present in the `Joi` delete creation config as well (in similar Python validator that would throw an error in some conditions).

Reply:
the router handler and Joi both validate the request input. This is an example of the serialized content returned in the response.

Reply:
Makes sense.  I thought that was request body which I now see the request doesn't require.